test/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ target_compile_definitions(test-sfml-system PRIVATE
     EXPECTED_SFML_VERSION_MAJOR=${SFML_VERSION_MAJOR}
     EXPECTED_SFML_VERSION_MINOR=${SFML_VERSION_MINOR}
     EXPECTED_SFML_VERSION_PATCH=${SFML_VERSION_PATCH}
-    EXPECTED_SFML_VERSION_IS_RELEASE=$<IF:$<BOOL:${VERSION_IS_RELEASE}>,true,false>
+    EXPECTED_SFML_VERSION_IS_RELEASE=$<BOOL:${SFML_VERSION_IS_RELEASE}>
```

Comment:
I intentionally avoided the shorter generator expression because I wanted the test output to show `true` or `false` instead of `1` or `0` to better express the boolean nature of this definition.

Reply:
I changed this because `cmakedefine01` turns things into `0` or `1`.